pkg/api/vlabs/const.go

```diff
@@ -134,6 +134,8 @@ const (
 const (
 	// AzureStackCloud is a const string reference identifier for Azure Stack cloud
 	AzureStackCloud = "AzureStackCloud"
+	// MaxAzureStackManagedDiskSize is max etcd disk size supported on AzureStackCloud
+	MaxAzureStackManagedDiskSize = 1023
```

Comment:
how does this change relate to https://github.com/Azure/aks-engine/pull/1558/files?

should this be `MaxAzureStackEtcdDiskSize` ?

Reply:
or I guess it's just the general limit imposed by disk RP, in which case the name makes sense

Reply:
Yes this is the general limit imposed by disk RP.